src/pyinit.jl

```diff
@@ -190,6 +190,24 @@ function __init__()
         Py_SetPythonHome(libpy_handle, pyversion, pyhome)
         Py_SetProgramName(libpy_handle, pyversion, current_python())
         ccall((@pysym :Py_InitializeEx), Cvoid, (Cint,), 0)
+
+        # _pydecref_queue used to pydecref only on the main thread:
```

Comment:
If someone calls Python from a different thread we are in trouble…

Reply:
Thats right, but at least with this we can say "you can run with threads>1, just dont call Python from a different thread" whereas the current situation is "running with threads>1 and PyCall may segfault even if you only call Python from the main thread". Or is there a scenario where a user which is careful to only call from the main thread may trigger calls from other threads? Anyway, not trying to push for this PR exactly, just wanted to point that out.